cns/types/types.go

```diff
@@ -909,3 +909,23 @@ type CnsVolumePolicyReconfigSpec struct {
 func init() {
 	types.Add("vsan:CnsVolumePolicyReconfigSpec", reflect.TypeOf((*CnsVolumePolicyReconfigSpec)(nil)).Elem())
 }
+
+type CnsSyncDatastore CnsSyncDatastoreRequestType
+
+func init() {
+	types.Add("vsan:CnsSyncDatastore", reflect.TypeOf((*CnsSyncDatastore)(nil)).Elem())
+}
+
+type CnsSyncDatastoreRequestType struct {
+	This         types.ManagedObjectReference `xml:"_this"`
+	DatastoreUrl string                       `xml:"datastoreUrl,omitempty"`
+	FullSync     string                       `xml:"fullSync,omitempty"`
```

Comment:
this is a bool field.

Reply:
chagne `xml:"fullSync,omitempty"` to `xml:"fullsync,omitempty"`

Reply:
Done.